components/pd_client/src/lib.rs

```diff
@@ -198,6 +198,15 @@ impl BucketStat {
         }
     }
 
+    pub fn ingest_sst(&mut self, key_count: u64, value_size: u64) {
+        for stat in self.stats.mut_write_bytes() {
+            *stat += value_size;
+        }
```

Comment:
```suggestion
        let first_idx = find_bucket_index(sst.first_key, &self.meta.keys);
        let last_idx = find_bucket_index(sst.last_key, &self.meta.keys);
        if last_idx - first_idx == 0 {
            if let Some(bytes) = self.stats.mut_write_bytes().get_mut(first_idx) {
                *bytes += value_size;
            }
        } else { 
            let avg = value_size/(last_idx - first_idx);
            for idx in first_idx..=last_idx {
                if let Some(bytes) = self.stats.mut_write_bytes().get_mut(idx) {
                    *bytes += avg;
                }
            }
        }
```

Maybe we should only add size to buckets that covers ingested ssts.

Reply:
no, the condition of data distribution in all the buckets is not true.